src/app/view/components/dot-iframe-dialog/dot-iframe-dialog.component.scss

```diff
@@ -2,6 +2,12 @@
 @import "variables";
 
 .dialog ::ng-deep {
+    .ui-dialog.ui-dialog-titlebar-animate {
+        .ui-dialog-titlebar {
+            transition: opacity 2s ease-in;
```

Comment:
2s is too much for this animation... the animation should be triggered when the text appear, there is a var in the scss for the animation speed.